test/service/test_request_validation.py

```diff
@@ -118,7 +124,8 @@ def test_bad_sort_facet_but_valid_filter_facet_of_sample(self):
         }
         response = requests.get(url, params=params)
         self.assertEqual(400, response.status_code, response.json())
-        self.assertEqual(self.facet_message, response.json())
+        self.assertEqual(self.facet_enumeration_message(),
+                         response.json())
```

Comment:
```suggestion
        self.assertEqual(self.facet_enumeration_message(), response.json())
```
A single line seems fine here